src/frontend/src/components/EditUser/index.tsx

```diff
@@ -186,7 +192,9 @@ class EditUser extends React.Component<Props, State> {
                         name="role"
                         value="search"
                         className="v-top"
-                        checked
+                        checked={this.state.role === "search"}
+                        onChange={this.handleRadioChange}
+                      />
```

Comment:
```suggestion

```

Shouldn't be here

Reply:
![hanging_bracket](https://user-images.githubusercontent.com/2104990/74788782-94379080-5267-11ea-99c4-8fde93d4acd2.png)

Reply:
I added an onClick to the pencil icon to redirect to the edit-user page. Since it was just one line I decided to go ahead and include it here.

Reply:
Cool.